website_docs/getting_started.md

```diff
@@ -77,11 +79,14 @@ Lastly, open a browser and navigate to the [Jaeger UI](http://localhost:16686) o
 
 Adding tracing to a single service is a great first step and although auto-instrumentation provides quite a bit of insight on its own, OpenTelemetry provides a few more features that will allow you gain even deeper insights!
 
+[Automatic Instrumentation][auto-instrumentation] .
+
+[Manual Instrumentation][manual-instrumentation] will give provide you the ability to enrich your traces with domain specific data.
```

Comment:
As I mentioned earlier, I think we want our users to be able to start using the SDK with the minimal amount of configuration or developer intervention so I recommend the following flow: 

- Context Propagation
- Manual Instrumentation
- Span Events
- Advanced Configuration Topics
   - Using OTEL environment variables 
   - Customizing automatic instrumentation 
   - Configuring Error Reporting
   - Performance Monitoring
   - etc...